test/unit/helpers/ExtensionHelperTest.php

```
@@ -22,49 +22,54 @@
         $this->assertEquals([$ext1,$ext2,$ext3], array_values($sorted));
     }
 
-    public function testCyclicDependencies()
+    public function testCyclicDependencies(): void
     {
         $this->expectException(common_exception_Error::class);
         $ext1 = $this->mockExtension('ext1', ['ext2']);
         $ext2 = $this->mockExtension('ext2', ['ext3']);
         $ext3 = $this->mockExtension('ext3', ['ext1']);
-        $sorted = \helpers_ExtensionHelper::sortByDependencies([$ext2,$ext3,$ext1]);
+        $sorted = helpers_ExtensionHelper::sortByDependencies([$ext2,$ext3,$ext1]);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sorted'. (undefined)

(UnusedLocalVariable)